app/src/interfaces/_system/system-theming/theme-color-picker/theme-color-picker.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+	<theme-color-picker :model-value="value" @update:model-value="$emit('input', $event)"></theme-color-picker>
+</template>
+
+<script lang="ts">
+export default {
+	name: 'ThemeColorPicker',
+};
+</script>
+
+<script lang="ts" setup>
+import { ValidationError } from '@directus/shared/types';
+import themeColorPicker from '../components/theme-color-picker.vue';
+
+interface Props {
+	value: string;
+	disabled?: boolean;
+	validationErrors?: ValidationError[];
+	field: string;
+	primaryKey: string | number;
+	batchMode?: boolean;
+	batchActiveFields?: string[];
+	loading?: boolean;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const props = withDefaults(defineProps<Props>(), {
```

Comment:
Oh wow... This is actually hilarious to me 😂 In all the time I spent thinking about how to handle this, not once did I think to just *not* assign it to a variable. In my mind I was thinking the setup syntax required a variable to export for use in the template. I have no clue why I didn't consider otherwise.

Thanks a bunch for catching this! I learned something here 😃